src/Umbraco.Core/Cache/Refreshers/Implement/UserCacheRefresher.cs

```diff
@@ -45,5 +45,18 @@ public override void Refresh(Guid id)
         base.Refresh(id);
     }
 
+    public override void Refresh(int id)
+    {
+        Remove(id);
+        base.Refresh(id);
+    }
+
+    public override void Remove(int id)
+    {
+        string cacheKey = $"uRepo_{typeof(IUser).Name}_" + id;
```

Comment:
Use RepositoryCacheKeys or CacheKeys here to construct this? Same below.

Reply:
Good point, done 👍 😁